feature/registration/src/main/java/com/jeanbarrossilva/orca/feature/registration/Registration.kt

```diff
@@ -0,0 +1,143 @@
+ * not, see https://www.gnu.org/licenses.
+ */
+
+package com.jeanbarrossilva.orca.feature.registration
+
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.scaleIn
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.jeanbarrossilva.orca.feature.registration.ui.stack.Stack
+import com.jeanbarrossilva.orca.feature.registration.ui.status.Status
+import com.jeanbarrossilva.orca.feature.registration.ui.status.StatusCard
+import com.jeanbarrossilva.orca.feature.registration.ui.status.rememberStatusCardState
+import com.jeanbarrossilva.orca.platform.animator.Animator
+import com.jeanbarrossilva.orca.platform.animator.animation.Motion
+import com.jeanbarrossilva.orca.platform.animator.animation.timing.after
+import com.jeanbarrossilva.orca.platform.autos.colors.asColor
+import com.jeanbarrossilva.orca.platform.autos.kit.action.button.PrimaryButton
+import com.jeanbarrossilva.orca.platform.autos.kit.scaffold.Scaffold
+import com.jeanbarrossilva.orca.platform.autos.kit.scaffold.bar.button.ButtonBar
+import com.jeanbarrossilva.orca.platform.autos.theme.AutosTheme
+import com.jeanbarrossilva.orca.platform.autos.theme.MultiThemePreview
+import kotlin.time.Duration.Companion.seconds
+
+@Composable
+private fun Registration(modifier: Modifier = Modifier, motion: Motion = Motion.Moving) {
+  val spacing = AutosTheme.spacings.large.dp
+  val spacerModifier = remember(spacing) { Modifier.height(spacing) }
+  val backdropColor =
+    if (isSystemInDarkTheme()) {
+      AutosTheme.colors.surface.container.asColor
+    } else {
+      AutosTheme.colors.placeholder.asColor
+    }
+  val statusCardAnimationSpec = remember { tween<Float>() }
+  val statusCardEnterTransition =
+    remember(statusCardAnimationSpec) {
+      fadeIn(statusCardAnimationSpec) + scaleIn(statusCardAnimationSpec, initialScale = .8f)
+    }
+  val statusCardDelay = remember { 2.seconds }
+
+  Scaffold(modifier, bottom = { ButtonBar { PrimaryButton(onClick = {}) { Text("Continue") } } }) {
+    expanded {
+      LazyColumn(
+        Modifier.padding(it).fillMaxHeight(),
```

Comment:
Padding coming from the scaffold should be set as the lazy column's content's instead of applied via a modifier.